RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3617,14 +3658,14 @@ boolean auto_change_mcd(int mcd, boolean immediately)
 			return false;
 		}
 	}
-	if($strings[Blender, Packrat, Wombat] contains my_sign())
+	if(inGnomeSign())
 	{
-		if(get_property("lastDesertUnlock").to_int() < my_ascensions())
+		if(gnomads_available())
 		{
 			return false;
 		}
 	}
-	if($strings[Marmot, Opossum, Platypus] contains my_sign())
+	if(inCanadiaSign())
 	{
 		best = 11;
 	}
```

Comment:
next line should use `in_bad_moon()` (stupid GitHub won't let me select that line even if I expand the diff).

Reply:
change made